src/game/mining/chunk.gd

```diff
@@ -0,0 +1,108 @@
+class_name Chunk
+extends Area2D
+
+@export var block_size_px: int = 20
+@export var radius: int = 3
+
+var has_spawened = false;
+
+var ordinal_positions: Array[Vector2] = [
+	Vector2.LEFT,
+	Vector2.RIGHT,
+	Vector2.UP,
+	Vector2.DOWN,
+	Vector2.LEFT + Vector2.UP,
+	Vector2.UP + Vector2.RIGHT,
+	Vector2.RIGHT + Vector2.DOWN,
+	Vector2.DOWN + Vector2.LEFT
+]
+
+@onready var chunk_size: int = 1 + (radius * 2)
+@onready var chunk_size_px: int = chunk_size * block_size_px
+
+@onready var chunk: PackedScene = preload("res://game/mining/chunk.tscn")
+@onready var area: CollisionShape2D = get_node("Area");
```

Comment:
get_node works fine, but there's actually a shortcut syntax you can use (`$Area` in this case).  Also, if you have a node that's deeply nested inside of other nodes (not that this one is), instead of using `$The/Full/Path/Area` you can right-click on it and select "% Access as Unique Name" and use `%Area` instead and that name will be a unique reference within that scene; especially nice when building UIs